openedx/core/djangoapps/programs/rest_api/v1/views.py

```diff
@@ -33,8 +33,8 @@ class Programs(APIView):
 
     permission_classes = (IsAuthenticated,)
 
-    def get(self, request: "HttpRequest", enterprise_uuid: str) -> "HttpResponse":
-        """For an enterprise learner, get list of enrolled programs with progress.
+    def get(self, request: "HttpRequest", enterprise_uuid: str = None) -> "HttpResponse":
```

Comment:
so, technically a `str` can't be none. But if at the top of the file, in the imports, you add `Optional` to the things imported from `typing`, then this can be:
```suggestion
    def get(self, request: "HttpRequest", enterprise_uuid: Optional[str] = None) -> "HttpResponse":
```

(a string could also default to empty, e.g. `enterprise_uuid: str = ''`, but I believe that making it  an optional string the defaults to `None`  is cleaner.)